doc/rbd/rbd-snapshot.rst

```diff
@@ -41,19 +44,22 @@ Cephx Notes
 
 When `cephx`_ authentication is enabled (it is by default), you must specify a
 user name or ID and a path to the keyring containing the corresponding key. See
-:ref:`User Management <user-management>` for details. You may also set the
-``CEPH_ARGS`` environment variable to avoid re-entry of these parameters. ::
+:ref:`User Management <user-management>` for details. 
+
+.. prompt:: bash $
+
+	rbd --id {user-ID} --keyring /path/to/secret [commands]
+	rbd --name {username} --keyring /path/to/secret [commands]
 
-	rbd --id {user-ID} --keyring=/path/to/secret [commands]
-	rbd --name {username} --keyring=/path/to/secret [commands]
+For example:
 
-For example:: 
+.. prompt:: bash $
 
 	rbd --id admin --keyring=/etc/ceph/ceph.keyring [commands]
 	rbd --name client.admin --keyring=/etc/ceph/ceph.keyring [commands]
```

Comment:
Need to get rid of `=` here too.

Reply:
I thought I got these. Anyway, I got them now.